common/src/main/java/com/microsoft/identity/common/adal/internal/net/IWebRequestHandler.java

```diff
@@ -60,4 +60,10 @@ HttpWebResponse sendPost(URL url, Map<String, String> headers, byte[] content,
      * @param requestCorrelationId {@link UUID} of the correlation id to set in the web request.
      */
     void setRequestCorrelationId(final UUID requestCorrelationId);
+
+    /**
+     * Set the version name of ADAL client
+     * @param adalClientVersion the version name of ADAL client
+     */
+    void setAdalClientVersion(String adalClientVersion);
```

Comment:
Rename to `setClientVersion` for eventual ADAL/MSAL sharing

Reply:
renamed. good catch